File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleops/arcade_drive/BlueArcadeDrive.java
package org.firstinspires.ftc.teamcode.teleops.arcade_drive;

import com.qualcomm.robotcore.eventloop.opmode.TeleOp;

import org.firstinspires.ftc.teamcode.Side;

@TeleOp(name = "Blue Arcade Drive", group = "Competition")
public class BlueArcadeDrive extends ArcadeDriveBase {
    @Override
    protected Side getSide() {
        return Side.BLUE;
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleops/arcade_drive/BlueArcadeDrive.kt
package org.firstinspires.ftc.teamcode.teleops.arcade_drive

import com.qualcomm.robotcore.eventloop.opmode.Disabled
import com.qualcomm.robotcore.eventloop.opmode.TeleOp
import org.firstinspires.ftc.teamcode.utils.Side

@TeleOp(name = "Blue Arcade Drive")
@Disabled
class BlueArcadeDrive : ArcadeDriveBase() {
    override val side: Side = Side.BLUE
}